apps/api-extractor/src/schemas/api-extractor.schema.json

```diff
@@ -68,8 +68,17 @@
         },
 
         "reportFileName": {
-          "description": "The filename for the API report files.  It will be combined with \"reportFolder\" or \"reportTempFolder\" to produce a full file path. The file extension should be \".api.md\", and the string should not contain a path separator such as \"\\\" or \"/\".",
-          "type": "string"
+          "description": "The filename for the complete (\"untrimmed\") API report files.  It will be combined with \"reportFolder\" or \"reportTempFolder\" to produce a full file path. The file extension should be \".api.md\", and the string should not contain a path separator such as \"\\\" or \"/\".",
+          "type": ["string", "null"]
```

Comment:
We don't ever use `null` in JSON files, due to a philosophical stance that `null` is [best avoided](https://github.com/microsoft/rushstack/tree/6398f7bf4440a82fb22eca3286519f5eef8d6438/eslint/eslint-plugin#rushstackno-new-null) in JavaScript code.

Reply:
Yup, happy to conform to practices followed by the repo.

My rationale here was that (IMO) introducing a special string value to avoid null is worse than using null. Likely moot in this particular case, but it reserves a potentially valid string value to mean something not captured by the type-system.